tox.ini

```diff
@@ -1,47 +1,53 @@
 [tox]
 envlist = py27,pypy,py33,py34,py35,flake8
+deps =
+    pytest
+    unicodecsv
+    hypothesis>=3.6.1,<4.0.0
+
 [testenv:py27]
 whitelist_externals = rm
-commands = 
+commands =
     rm -f jellyfish/cjellyfish.so
     pip install -e .
-    py.test jellyfish/test.py
-deps = pytest
-    unicodecsv
+    pytest jellyfish/test.py jellyfish/test_properties.py
+deps = {[tox]deps}
+
 [testenv:pypy]
-commands = 
+commands =
     pip install -e .
-    py.test jellyfish/test.py
-deps = pytest
-    unicodecsv
+    pytest jellyfish/test.py jellyfish/test_properties.py
+deps = {[tox]deps}
+
 [testenv:py33]
 whitelist_externals = rm
-commands = 
+commands =
     rm -f jellyfish/cjellyfish.so
     pip install -e .
-    py.test jellyfish/test.py
-deps = pytest
-    unicodecsv
+    pytest jellyfish/test.py jellyfish/test_properties.py
+deps = {[tox]deps}
+
 [testenv:py34]
 whitelist_externals = rm
-commands = 
+commands =
     rm -f jellyfish/cjellyfish.so
     pip install -e .
-    py.test jellyfish/test.py
-deps = pytest
-    unicodecsv
+    pytest jellyfish/test.py jellyfish/test_properties.py
+deps = {[tox]deps}
+
 [testenv:py35]
 whitelist_externals = rm
-commands = 
+commands =
     rm -f jellyfish/cjellyfish.so
     pip install -e .
-    py.test jellyfish/test.py
-deps = pytest
-    unicodecsv
+    pytest jellyfish/test.py jellyfish/test_properties.py
+deps = {[tox]deps}
+
 [testenv:flake8]
-basepython=python2.7
+basepython = python2.7
```

Comment:
I've added whitespace around the `=` to make it consistent with the rest of the file.